src/Formula/Parsing.hs

```diff
@@ -62,13 +67,6 @@ notFollowedByElse :: Parser a -> (a -> Parser ()) -> Parser ()
 notFollowedByElse p f = try ((try p >>= f) <|> pure ())
 
 
-
-class Parse a where
-  parser :: Parser a
-  default parser :: FromGrammar a => Parser a
-  parser = formulaParser
-
-
```

Comment:
This change is not so innocent. It will break the Autotool binding, which expects to find the `Parse` class inside `LogicTasks.Parsing` which itself is just a reexport of `Formula.Parsing`.

Reply:
I reverted that.